massgov/openmass
# DP-23393: Back and front end work for breadcrumbs v1 (Deploy 2)

**The ticket mentions to NOT merge this into the integration branch. There will also be mayflower changes that need to merge prior to this PR.**

**Description:**
Added the breadcrumb to page templates and removed 'Part of' from node templates.


**Jira:** (Skip unless you are MA staff)
DP-23393


**To Test:**
- Login and view content of the following types:
- Advisory
- Binder
- Curated List
- Decision
- Decision Tree
- Event
- Executive Order
- Form
- Guide
- How-to
- Information Details
- Location
- Location Details
- News
- Organization
- Person
- Promo Page
- Regulation
- Rules of Court
- Service
- Service Details
- Topic
- Verify the breadcrumb is displaying.
- Verify "Part of" is no longer displaying.
- Verify "Offered by" is still showing for all except Organization and Promo Page.


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
